site/content/docs/5.2/components/breadcrumb.md

```diff
@@ -106,6 +124,20 @@ As part of Boosted's evolving CSS variables approach, breadcrumbs now use local
 
 {{< scss-docs name="breadcrumb-css-vars" file="scss/_breadcrumb.scss" >}}
 
+<!-- Boosted mod: Dark variant -->
+Customization through CSS variables can be seen on the `.breadcrumb-dark` modifier class where we override specific values without adding duplicate CSS selectors.
+
+{{< scss-docs name="breadcrumb-dark-css-vars" file="scss/_breadcrumb.scss" >}}
+<!-- End mod -->
+
 ### Sass variables
 
+Variables for all breadcrumbs:
```

Comment:
Should maybe be in Bs (in all files)

Reply:
Let's see the feedback from https://github.com/twbs/bootstrap/pull/37034